04_Scheduling/03_Tasks_And_Threads.md

```diff
@@ -62,13 +62,109 @@ typedef struct
 
 Where `NAME_MAX_LEN` is a constant with the maximum length for our process it´s up to us. The pid is a unique number, it can even be a uuid if we want, but the easiest way to achieve this is just using sequential number that starts from zero. To achive this we just need to add a global variable that contains the next available number. Now since we are using size_t on a 64bit architecture, we don't need to worry about overflowing the variable size, technically we can decide to reuse pids that are no longer used by a process, that is totally up to us, but in our simple scenario we will increment a global variable: 
 
-```
+```c
 size_t next_free_pid = 0;
 ```  
 
+
 ### Virtual memory space
 
-TBD 
+On of the most useful features of having the paging enabled is that we have the concept of Virtual Memory available (for a complete definition refer to the [Paging](,,/02_Memory_Management/Paging.md) and [Virtual Memory](../02_Memory_Management/04_Virtual_Memory_Manager.md) chapter), and from a process point of view it means that we can have each process with it's own addressing space. 
```

Comment:
`Virtual Memory` is not a title or proper noun so it doesn't get capitalized here (there are other examples of this, I'm just using these two).